include/psa/crypto_extra.h

```diff
@@ -1901,6 +1901,342 @@ static inline struct psa_pake_operation_s psa_pake_operation_init( void )
     return( v );
 }
 
+/**
+ * \brief The context for PSA interruptible signing.
+ *
+ * \note  Contents not yet designed as implementation specific.
+ *
+ */
+struct psa_sign_interruptible_context_s
+{
+    size_t MBEDTLS_PRIVATE(num_ops);
+};
+
+typedef struct psa_sign_interruptible_context_s psa_sign_interruptible_context;
```

Comment:
Btw how am I supposed to initialize this? Don't we need the usual set of initialization tools (macro, function)?